### PR TITLE
Added test for SimpleReflexAgentProgram

### PR DESCRIPTION
**Pull request by Noumanmufc1**
I have added the test for `SimpleReflexAgentProgram` in agents.py. @MrDupin  Please review?

### CODE REVIEW

**Comment by ad71** [comment]
Don't forget to update README.md

**Comment by Noumanmufc1** [comment]
Done. Thanks @ad71

File: tests/test_agents.py [hunk 1]
```diff
@@ -156,6 +157,38 @@ def test_TableDrivenVacuumAgent() :
...
+    loc_B = (1, 0)
+    
+    # create rules for a two state Vacuum Environment
+    rules = [Rule((loc_A, "Dirty"), "Suck"), Rule((loc_A, "Clean"), "Right"),
+            Rule((loc_B, "Dirty"), "Suck"), Rule((loc_B, "Clean"), "Left")]
+    
+    def interpret_input(state):
+        return state
+    
+    # create an program and then an object of the SimpleReflexAgentProgram
```
Comment (MrDupin) [inline tests/test_agents.py]: It's 'create a program'.

**Comment by Noumanmufc1** [comment]
Done with the typo.

**Pull request merged**